src/components/progress-tracker/progress-tracker-test.stories.tsx

```diff
@@ -91,3 +97,26 @@ LeftLabelWithLabelWidth.parameters = {
 export const ProgressTrackerComponent = (props: ProgressTrackerProps) => {
   return <ProgressTracker progress={50} showDefaultLabels {...props} />;
 };
+
+export const InsideFlexTile = () => {
+  return (
+    <Tile m={0} py={0}>
+      <FlexTileContainer>
+        <FlexTileCell py={2}>
+          <FlexTileDivider />
+          <ProgressTracker
+            length="100%"
+            progress={50}
+            currentProgressLabel="Step 3"
+            maxProgressLabel="5"
+          />
+        </FlexTileCell>
+      </FlexTileContainer>
+    </Tile>
+  );
+};
+
+InsideFlexTile.storyName = "inside flex tile";
+InsideFlexTile.parameters = {
+  themeProvider: { chromatic: { disableSnapshot: false } },
+};
```

Comment:
I have done this intentionally as we want to snapshot this to make sure we catch any future regressions.